doc/src/manual/getting-started.md

```diff
@@ -65,19 +66,19 @@ foo
 bar
 ```
 
-The `--` delimiter can be used to separate command-line args to the scriptfile from args to Julia:
+The `--` delimiter can be used to separate command-line arguments for the script file from arguments for Julia:
```

Comment:
Arguments `to` the file sounds good to me?

Reply:
I clarified my intentions a bit... :)